pkg/cluster/inventory_test.go

```diff
@@ -314,7 +317,7 @@ func TestCountRetries(t *testing.T) {
 			require.NoError(t, err)
 		}
 		//count how often retry happened
-		cnt, err := inventory.CountRetries(clusterState.Configuration.RuntimeID, clusterState.Configuration.Version)
+		cnt, err := inventory.CountRetries(clusterState.Configuration.RuntimeID, clusterState.Configuration.Version, 250)
```

Comment:
If you put `250`, the limit number is `250 * 5`, is it the actual expectation?

Reply:
Yes, since we want maximum of 250 retires in this test case ( we can also lower it, doesnt matter tbh), then we need something around 5 times the rows of the db table

Reply:
But indeed I will put it to a lower number, makes more sense